### PR TITLE
fix(unify): Remove 'Reconfigure' dropdown from Testing Type chooser

### PR DESCRIPTION
**Pull request by davidmunechika**
- Closes [UNIFY-1533](https://cypress-io.atlassian.net/browse/UNIFY-1533?atlOrigin=eyJpIjoiYjg0YjJkZjYzNmUyNDUwOThhYmJiZGM0NzNkNjJmNTYiLCJwIjoiaiJ9)

### User facing changelog
Remove the ‘Reconfigure’ dropdown from testing type chooser.
Update: this also removes the 'Choose a Browser' dropdown from testing type chooser since the functionality of this doesn't work as intended.

### Additional details

Previous version:

Screenshot 2022-04-13 at 10 49 35 AM

Updated version:

Screenshot 2022-04-13 at 10 42 55 AM

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [n/a] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
@davidmunechika It looks like this percy screenshot is still showing a dropdown arrow on the E2E side. https://percy.io/cypress-io/cypress/builds/17329531/changed/972768022?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=800&widths=800%2C1000

File: packages/frontend-shared/src/gql-components/StatusBadge.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
           />
           {{ titleOn }}
           <i-cy-chevron-down-small_x8
+            v-if="!props.isRunning"
```
Comment (ryanthemanuel) [inline packages/frontend-shared/src/gql-components/StatusBadge.vue:15]: Would it make more sense for this to be:

```suggestion
            v-if="menuItems.length > 0"
```

Functionally the same today but might future proof things a bit.

**Comment by ryanthemanuel** [comment]
This is maybe out of scope for this change, but I was playing around with the state of things now and the drop down we get when we're selecting a testing type from the app is a little weird.

If you click the drop down you get an option to select 'Choose a Browser'

image

However, if you click that, it doesn't take you to the "Choose a Browser" screen, it launches the testing type you selected with the same browser.

**Comment by davidmunechika** [comment]
> This is maybe out of scope for this change, but I was playing around with the state of things now and the drop down we get when we're selecting a testing type from the app is a little weird.
> 
> If you click the drop down you get an option to select 'Choose a Browser'
> 
> image
> 
> However, if you click that, it doesn't take you to the "Choose a Browser" screen, it launches the testing type you selected with the same browser.

@ryanthemanuel Hmm... should I just remove the dropdown altogether?

File: packages/frontend-shared/src/gql-components/StatusBadge.vue (hunk 1 above)
Comment (lmiller1990) [reply packages/frontend-shared/src/gql-components/StatusBadge.vue:15]: Better yet, it's not even a list of items anymore, so we can just

```html
<MenuItems v-if="!props.isRunning">
  <MenuItem>
    <!-- stuff -->
```

No need to use a list when you either have 1 item or 0 items.

**Review by lmiller1990 — COMMENTED** [review]
I think we can greatly simplify this, now we do not have a list, but a single item (effectively a boolean... `if (!props.isRunning) { /* thing */ }`.

**Comment by ZachJW34** [comment]
@davidmunechika I can't find the mocks in Figma for this dropdown, and seeing that it doesn't work as intended I say we remove it from the TestingTypeSwitcher. Would be good to sync with @ryanjwilke, but for this PR we can remove the menu and you can create a followup issue for adding back in the `Choose a Browser` menu item if it's still desired.

**Comment by lmiller1990** [comment]
@davidmunechika I see you raised this internally and clarified the goal. Is this PR ready for re-review?

**Comment by davidmunechika** [comment]
@lmiller1990 Yes, sorry- was waiting for the build to finish and forgot to re-request reviewers

**Comment by ryanjwilke** [comment]
@davidmunechika We can safely remove the "Choose a browser" action and the entire menu in this PR. That should be fine. This is not something we'll need to add back in.

**Review by ZachJW34 — APPROVED** [review]

**Comment by ZachJW34** [comment]
@davidmunechika can you create an issue for cleaning up the backend code related to `reconfigure project`?

**Comment by davidmunechika** [comment]
@ZachJW34 Created one here: https://cypress-io.atlassian.net/browse/UNIFY-1593

File: packages/frontend-shared/src/gql-components/StatusBadge.vue [hunk 2]
```diff
@@ -67,44 +32,4 @@ const props = defineProps<{
...
-    currentProject {
-      id
-      currentTestingType
-      isCTConfigured
-      isE2EConfigured
-      isLoadingConfigFile
-      isLoadingNodeEvents
-    }
-  }
-}
```
Comment (estrada9166) [inline packages/frontend-shared/src/gql-components/StatusBadge.vue:88]: Should we remove the code on the graphql side as well? it seems that the mutation `setTestingTypeAndReconfigureProject` is only being used from this component

**Pull request merged**